eng/SourceBuild.props

```diff
@@ -16,6 +16,10 @@
           DependsOnTargets="PrepareInnerSourceBuildRepoRoot"
           BeforeTargets="RunInnerSourceBuildCommand">
     <Delete Files="$(InnerSourceBuildRepoRoot).globalconfig" />
+
+    <PropertyGroup>
+      <TargetRid>$(TargetRid)</TargetRid>
```

Comment:
I don't understand. Doesn't this do nothing?

Reply:
I renamed the property to match and must have forgotten to update this. I'll fix it.